ParisBNF/abb/BNFabb71.xml

```diff
@@ -0,0 +1,226 @@
+         </titleStmt>
+         <publicationStmt>
+            <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+            <pubPlace>Hamburg</pubPlace>
+            <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+               Forschungsumgebung / Beta maṣāḥǝft</publisher>
+            <availability>
+               <licence target="http://creativecommons.org/licenses/by-sa/4.0/">
+                  <p> This file is licensed under the Creative Commons Attribution-ShareAlike 4.0.
+                  </p>
+               </licence>
+            </availability>
+
+         </publicationStmt>
+         <sourceDesc>
+            <msDesc xml:id="ms">
+               <msIdentifier>
+                  <repository ref="INS0303BNF"/>
+                  <collection>Manuscrits orientaux</collection>
+                  <collection>Fonds éthiopien</collection>
+                  <collection>d'Abbadie</collection>
+                  <idno facs="https://gallica.bnf.fr/ark:/12148/btv1b525214715">BnF Éthiopien d'Abbadie 71</idno>
+               </msIdentifier>
+               <msContents>
+                  <summary/>
+
+                  <msItem xml:id="ms_i1">
+                     <locus from="1ra" to="89vb"/>
+                     <title type="complete" ref="LIT2342serata">The Order of the Monastic Profession</title>
+                     <textLang mainLang="gez"/>
+                     <incipit xml:lang="gez"><hi rend="rubric">በስመ፡ አብ፡ ወወልድ፡ <gap reason="ellipsis"
+                              resp="DN"/></hi> ዝንቱ፡ ሥርዓተ፡ መንኵስና፡ ዘሠርዓ፡ አቡነ፡ ቅዱስ፡ <hi rend="rubric"
+                           >አባ፡ ያዕቆን፡ ዘስሩግ። ጸሎቱ፡ ወበረከቱ፡ የሀሉ፡ ምስ</hi>ለ፡ ፍቁሩ፡ <persName ref="PRS14540HendaKrestos">ሕንፃ፡ ክርስቶስ፡</persName> ለዓለመ፡ ዓለም፨ ይደሉ፡ ለዘይፈቀድ፡
+                        ይኩን፡ መነኮስ፡ ከመ፡ ይንበር፡ ዓመተ፡ ይትመሀር፡ ውሳንያተ፡ መንኵስና፡ ወይትገሠጽ፡ እመጻሕፍተ፡ ገነት፡ ዘአባዊነ፡
+                     </incipit>
+                  </msItem>
+
+                  <msItem xml:id="ms_i2">
+                     <locus from="90ra" to="91vb"/>
+                     <title type="complete" ref="LIT4498Pilgrims">Antiphon of Pilgrims</title>
+                     <textLang mainLang="gez"/>
+                     <incipit xml:lang="gez"><hi rend="rubric">መዝሙር፡ ዘመነኮሳት፡ ወፈለስያ</hi>ን፨ በ፫ወቅኑታን፡ ሐቌሆሙ፡ በጽድ<hi rend="rubric">ቅ፡ 
+                        ጠለ፡ ገዳም፡ ይሴሰዩ፡ ምስለ፡ እንስ</hi>ሳ፡ ገዳም፡ ይነብሩ፡ </incipit>
+                     <notatedMusic>
+                        <desc>Musical notation throughout the text.</desc>
+                     </notatedMusic>
+                  </msItem>
+
+               </msContents>
+               <physDesc>
+                  <objectDesc form="Codex">
+                     <supportDesc>
+                        <support>
+                           <material key="parchment"/>
+                        </support>
+                        <extent>
+                           <measure unit="leaf" quantity="79">4+91+3</measure>
```

Comment:
something does not match. quantity 98? or 4+72+3?

Reply:
done